tests/bfd/test_bfd.py

```
@@ -105,72 +142,104 @@ def stop_ptf_bfd(ptfhost):
 
 
 def add_ipaddr(ptfhost, neighbor_addrs, prefix_len, neighbor_interfaces, ipv6=False):
+    cmd_buffer = ""
     for idx in range(len(neighbor_addrs)):
         if ipv6:
-            ptfhost.shell("ip -6 addr add {}/{} dev eth{}".format(neighbor_addrs[idx], prefix_len, neighbor_interfaces[idx]))
+            cmd_buffer += "ip -6 addr add {}/{} dev eth{} ;".format(neighbor_addrs[idx], prefix_len, neighbor_interfaces[idx])
         else:
-            ptfhost.shell("ip addr add {}/{} dev eth{}".format(neighbor_addrs[idx], prefix_len, neighbor_interfaces[idx]))
-
+            cmd_buffer += "ip addr add {}/{} dev eth{} ;".format(neighbor_addrs[idx], prefix_len, neighbor_interfaces[idx])
+        if idx%50 == 0:
+            ptfhost.shell(cmd_buffer)
+            cmd_buffer = ""
+    if cmd_buffer != "":
+        ptfhost.shell(cmd_buffer)
 
 def del_ipaddr(ptfhost, neighbor_addrs, prefix_len, neighbor_interfaces, ipv6=False):
+    cmd_buffer = ""
     for idx in range(len(neighbor_addrs)):
         if ipv6:
-            ptfhost.shell("ip -6 addr del {}/{} dev eth{}".format(neighbor_addrs[idx], prefix_len, neighbor_interfaces[idx]), module_ignore_errors=True)
+            cmd_buffer += "ip -6 addr del {}/{} dev eth{} ;".format(neighbor_addrs[idx], prefix_len, neighbor_interfaces[idx])
         else:
-            ptfhost.shell("ip addr del {}/{} dev eth{}".format(neighbor_addrs[idx], prefix_len, neighbor_interfaces[idx]), module_ignore_errors=True)
-
+            cmd_buffer += "ip addr del {}/{} dev eth{} ;".format(neighbor_addrs[idx], prefix_len, neighbor_interfaces[idx])
+        if idx%50 == 0:
+            ptfhost.shell(cmd_buffer)
+            cmd_buffer = ""
+    if cmd_buffer != "":
+        ptfhost.shell(cmd_buffer,  module_ignore_errors=True) 
```

What's the purpose of ignoring errors here and in other places?